backend/src/controllers/reports/Tab_20_rpt_PA_Billed.js

```diff
@@ -18,7 +18,6 @@ controller.getReport = getReport;
  * @returns {object}
  */
 controller.Tab_20_rpt_PA_Billed = async (request, reply) => {
-  controller.userRequires(request, "PMO-Reports-Capability", reply);
```

Comment:
needs this role added to the route config.role

Reply:
It's in the route already, thanks to you!